scripts/convert_kakao_brain_unclip_to_diffusers.py

```diff
@@ -564,6 +564,7 @@ def super_res_unet_last_step_original_checkpoint_to_diffusers_checkpoint(model,
 
 # unet utils
 
+
 # <original>.time_embed -> <diffusers>.time_embedding
```

Comment:
New `black`? :thinking:

Reply:
Yeah, updated my black and then re-updated back to 22.8. We should maybe soon blackify the complete codebase once :-)